builtin/credential/approle/path_role.go

```diff
@@ -2355,9 +2375,28 @@ func (b *backend) handleRoleSecretIDCommon(ctx context.Context, req *logical.Req
 		return nil, err
 	}
 
+	var numUses int
+	// Check whether or not num_uses is defined, otherwise fallback to secret_id_num_uses
+	if numUsesRaw, ok := data.GetOk("num_uses"); ok {
+		if numUses < 0 {
```

Comment:
Please move this check to line 2388, as this is not doing much here.

Reply:
I have moved it to line 2382 instead. This check is only to validate the supplied field, not the role's configuration. Moved it instead of creating a second duplicate variable on the stack from `numUsesRaw`.